contracts/index.ts

```diff
@@ -1,2 +0,0 @@
-export { UserOperationStruct } from './types/EntryPoint'
```

Comment:
true, should be removed (along with the code to use it during "yarn publish"), but unrelated to TokenPaymaster.

Reply:
This was messing up the environment for me though.